sys-fs/bees/bees-9999.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit multilib linux-info
+
+DESCRIPTION="Best-Effort Extent-Same, a btrfs dedup agent"
+HOMEPAGE="https://github.com/Zygo/bees"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/kakra/bees.git"
+	EGIT_BRANCH="integration"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Zygo/bees/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+PATCHES="${FILESDIR}/v0.5-gentoo_build.patch"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="tools test"
```

Comment:
I don't see you using `test` flag anywhere.

Reply:
Test is running implicitly through `src_test()` by running `make test`, doesn't it? The build system supports it. And it's working when I run `ebuild bees-9999.ebuild test`.

But I just found out I need to remove the no-op check target from the Makefile, fixed it.

Reply:
You don't need `IUSE=test` for that. The USE flag is only necessary when you need to enforce extra dependencies or have extra conditionals on `use test`.